web/src/cypress/integration/rotations.ts

```diff
@@ -252,7 +263,9 @@ function testRotations(screen: ScreenFormat) {
         .clear()
         .type('5')
 
-      cy.get('button[type=submit]').click()
+      cy.get('button[type=submit]')
+        .eq(1)
```

Comment:
Determining which submit button based off of order within the DOM can be fragile in future tests if some code ever changes, which could cause regressions in the future. I would suggest adding a `data-cy=''` attribute to the submit button you're trying to press, and in cypress code you can reference it by `cy.get('button[data-cy="my-attribute-name"]').click()`, which will guarantee that it will always be the correct submit button clicked on.